netlink-proto/src/protocol/request.rs

```diff
@@ -5,36 +5,19 @@ use netlink_packet_core::NetlinkMessage;
 use crate::sys::SocketAddr;
 
 #[derive(Debug)]
-pub struct Request<T, M>
+pub enum Request<T, M>
```

Comment:
This is breaking API. Please increase the version number of `netlink-proto` in this PR.

The `enum/struct Request` is not used outside of `netlink-proto`, please convert it to `pub(crate)` if possible.

Reply:
I converted this to `pub(crate)` in a separate commit now, but don't think it breaks the API - I don't think the types (`Protocol`, `Request`, `Response`) were accessible from other crates before.